gradle.properties

```diff
@@ -12,7 +12,12 @@
 android.enableJetifier=true
 android.useAndroidX=true
 
-org.gradle.jvmargs=-Xmx4608M -XX:+UseParallelGC
+org.gradle.jvmargs=-Xmx4608M -XX:+UseParallelGC \
```

Comment:
why?

Reply:
required to run spotlessApply on jdk 17 in local